packages/traceloop-sdk/traceloop/sdk/experiment/experiment.py

```
@@ -39,7 +40,7 @@ async def run(
         task: Callable[[Optional[Dict[str, Any]]], Awaitable[Dict[str, Any]]],
         dataset_slug: Optional[str] = None,
         dataset_version: Optional[str] = None,
-        evaluators: Optional[List[EvaluatorDetails]] = None,
+        evaluators: List[EvaluatorSpec] = None,
```

Ruff (0.14.7)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)